src/status_im/components/styles.cljs

```diff
@@ -20,6 +20,7 @@
 (def color-gray6 "#212121")
 (def color-gray7 "#9fa3b4")
 (def color-gray8 "#6E777E")
+(def color-gray9 "#E9EBEC")
```

Comment:
We have a serious gray problem !!

Reply:
yes probably maybe we should report that to the design team, but here I just used the color that was in zepplin and it wasn't already there